core/system/inc/linker.h

```diff
@@ -65,6 +65,10 @@ typedef struct {
     /* address of __uvisor_box_context */
     uint32_t **uvisor_box_context;
 
+    /* heap for page allocator */
```

Comment:
Can you replace this comment with something like `/* Page allocator heap boundaries and size */`? See [this commit](https://github.com/ARMmbed/uvisor/pull/233/commits/d1128a81846fed3f584b8677bbd46393a8ae651d) for reference.